### PR TITLE
escalate major name changes to features only above v10

### PR DESCRIPTION
**Pull request by geohacker**
cc @bkowshik @manoharuss @krishnanammala @maning

### CODE REVIEW

File: tests/fixtures/major-name-modification.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
         "type": "Feature",
         "properties": {
           "name": "Bengaluru",
-          "wikidata": "Q40435"
+          "wikidata": "Q40435",
+          "osm:version": 9
```
Comment (bkowshik) [inline tests/fixtures/major-name-modification.json]: @geohacker Should the version be `> 10` to make sure we check just the name modification and version wise things are good here: https://github.com/mapbox/osm-compare/blob/master/comparators/major-name-modification.js#L21

File: tests/fixtures/major-name-modification.json [hunk 2]
```diff
@@ -27,7 +28,8 @@
         "type": "Feature",
         "properties": {
           "name": "Bangaluru",
-          "wikidata": "Q40435"
+          "wikidata": "Q40435",
+          "osm:version": 9
```
Comment (bkowshik) [inline tests/fixtures/major-name-modification.json]: @geohacker Should the version be > 10 to make sure we check just the name modification and version wise things are good here: https://github.com/mapbox/osm-compare/blob/master/comparators/major-name-modification.js#L21

File: tests/fixtures/major-name-modification.json [hunk 3]
```diff
@@ -50,10 +52,35 @@
         "type": "Feature",
         "properties": {
           "name": "BANGALURU",
+          "wikidata": "Q40435",
+          "osm:version": 9
```
Comment (bkowshik) [inline tests/fixtures/major-name-modification.json]: @geohacker Should the version be > 10 to make sure we check just the name modification and version wise things are good here: https://github.com/mapbox/osm-compare/blob/master/comparators/major-name-modification.js#L21

File: tests/fixtures/major-name-modification.json [hunk 4]
```diff
@@ -50,10 +52,35 @@
...
         },
         "geometry": null
       },
+      "expectedResult": {
+        "result:levenshtein_distance": 8,
+        "result:major_name_modification": true
+      }
+    },
+    {
+      "description": "Test minor name modification with case",
```
Comment (bkowshik) [inline tests/fixtures/major-name-modification.json]: There is already a test case with the description, `Test minor name modification with case`, shall we modify this to `Test major name modification with version > 10`

File: tests/fixtures/major-name-modification.json (hunk 1 above)
Comment (geohacker) [inline tests/fixtures/major-name-modification.json]: @bkowshik I added another fixture for that: https://github.com/mapbox/osm-compare/blob/f3cfa0c8e59620546f3d26992a334378049a06d9/tests/fixtures/major-name-modification.json#L98-L122

File: tests/fixtures/major-name-modification.json (hunk 4 above)
Comment (geohacker) [inline tests/fixtures/major-name-modification.json]: @bkowshik oops, yes. I didn't update the description of the tests :-/

**Pull request merged**